### ISSUE TITLE
Custom scripts per Profile tab

### ISSUE THREAD

**Issue by oswee** [gnunn1/tilix#1590]
When multi-tab Profile are loaded I would like to execute custom commands automatically on every Profile Tab.
I would like to have some kind of option to define commands in `profile.json` file.
It is important that Tabs are loaded in sequence of Tab numbers... 1st Tab + Execution; 2nd Tab + Execution and so on. It is not mandatory to wait while previous Tab finishes its execution, just open+execute tabs in their sequence.
Basically like Terminator Layout Custom commands.

**Comment by gnunn1** [comment]
You can do this already assuming I'm understanding correctly. Tilix supports saving the current layout, including profiles, into a file that can be loaded later as a session. You can set the command you want to run when the session using the "Layout Options" menu item. The command you set here gets saved into the session file and the command will be executed when you load the session in the future.

**Comment by gnunn1** [comment]
No response so I'm assuming my response answered the question, closing issue.
